proposals/description/tip_vebalotc.ts

```diff
@@ -62,23 +62,30 @@ const tip_vebalotc: TemplatedProposalDescription = {
       target: 'proxyAdmin',
       values: '0',
       method: 'upgrade(address,address)',
-      arguments: (addresses) => [addresses.balancerGaugeStaker, addresses.veBoostManagerImplementation],
+      arguments: (addresses) => [addresses.balancerGaugeStakerProxy, addresses.balancerGaugeStakerV2Impl],
       description: `Upgrade implementation of the balancerGaugeStaker`
     },
     {
-      target: 'veBoostManager',
+      target: 'balancerGaugeStaker',
```

Comment:
I don't think this is correct. 

You need to be calling `balancerGaugeStakerV2Impl.initialize()` rather than reinitialise the original implementation.

`balanceGaugeStakerV2Impl` inherits from `BalanceGaugeStaker` and only defines new storage variables later, ensuring there are no storage collisions. You need to initialise the new implementation as there is a new storage variable

Reply:
The proxy will already have it's originally defined V1 state variables initialised. We now need to initialise the new V2 state variable

Reply:
After the upgrade, the proxy has BalancerGaugeStakerV2 implementation. We need to call the new `_initialize` function

Reply:
Yes you do, but you are calling the V1 initialise function here

Reply:
No. In order to call the V2 init function we need to call `_initialize()`.